src/screens/ChannelListScreen/index.tsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable no-use-before-define */
+import * as React from 'react';
+import {FlatList} from 'react-native';
+
+import CommunityChannelItem from '../../components/ChatList/CommunityChannelItem';
+import GroupChatChannelItem from '../../components/ChatList/GroupChatChannelItem';
+import MessageChannelItem from '../../components/AnonymousChat/MessageChannelItem';
+import PostNotificationChannelItem from '../../components/AnonymousChat/PostNotificationChannelItem';
+import useFollowUser from './hooks/useFollowUser';
+import useSignedChannelListScreenHook from '../../hooks/screen/useSignedChannelListHook';
+
+const ChannelListScreen = () => {
+  const {channels, goToChatScreen, goToPostDetailScreen, goToCommunityScreen} =
+    useSignedChannelListScreenHook();
+  const {handleFollow, isInitialFollowing} = useFollowUser();
+
+  return (
+    <FlatList
+      data={channels}
+      keyExtractor={(item) => item.id}
+      scrollEnabled={false}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Setting `scrollEnabled` to `false` on a `FlatList` can lead to poor user experience if the list grows beyond the viewport. Consider enabling scrolling.

```diff
- scrollEnabled={false}
+ scrollEnabled={true}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
not needed as the parent can be scrolled